New style: Gold Leaf

rshatch opened the pull request:
Fixes #684 - one new style and color theme  by rosecarmine

rshatch commented:
(I went and fixed the issues 90d mentioned in her comment on the entry)

kareila commented on styles/goldleaf/layout.s2:
```diff
@@ -0,0 +1,2500 @@
+
+@media $medium_media_query {
+    #header .inner{
+        width: $*topbar_width_size$*topbar_width_units;
```
This evaluates to "width: ;" on my dreamhack, which means $*topbar_width_size and $*topbar_width_units are blank somehow?  Now that I look again at the same layer on DW that you linked me to earlier, it looks like it had the same problem there as well.

kareila commented on styles/goldleaf/layout.s2:
```diff
@@ -0,0 +1,2500 @@
+
+@media $medium_media_query {
+    #secondary > .inner:first-child{
+        width: $*topbar_width_size$*topbar_width_units;
```
Same problem here.

kareila commented on styles/goldleaf/layout.s2:
```diff
@@ -0,0 +1,2500 @@
+}
+
+@media $medium_media_query {
+    ul.userlite-interaction-li
```
Same problem here with the bottombar variables.

kareila replied:
Same problem here.

kareila commented:
I love this style but I did notice a couple of problems trying to load it on my dreamhack.  One is that the bottombar_width and topbar_width variables don't seem to have been given values - I pointed out in the stylesheet where I was seeing errors related to that.  The other problem I'm seeing is that the related images for tag, bullet, etc. aren't linked correctly.  Example link: http://system.kareila.hack.dreamwidth.net/res/4/goldleaf/elegantnotebook_groups.png - shouldn't that be .../styles/goldleaf/...?

If you can sort this out in the next couple of days I'll try to get it in for the code push :)

rshatch commented:
The URLs in the theme file were right, but they weren't being passed through a generator function. And I added default widths for the top and bottom bars, which uncovered some issues with content box sizing, so I fixed those too.

kareila commented:
I'm still seeing one error, near the bottom of the generated HTML:
```html
<img title="Top of page" alt="Top of page" src="goldleaf/elegantnotebook_top.png">
```
That image src is returning 404.
